### ISSUE TITLE
Can't create PV from line region

### ISSUE THREAD

**Issue by keflavich** [CARTAvis/carta#142]
**Describe the bug**
The "Generate" button is 'greyed out' on the PV slicer tool

**To Reproduce**

![image](https://user-images.githubusercontent.com/143715/169548118-bf34e872-6b24-4cf9-a239-a011dddc2139.png)

After closing all of the matched images, I was able to get a PV diagram - so it might have something to do with matching?  Perhaps another checklist item needs to be added?

**Platform info (please complete the following information):**
 - OS: rh7
 - Chrome
 - 3.0b3

**Comment by kswang1029** [comment]
@keflavich did you generate a pv image once but close it manually? If you redefine the line region you should be able to generate a new pv image.

**Comment by keflavich** [comment]
No, this was my first attempt to generate a PV image, and I did try several times to delete and re-create the line region - same result

**Comment by kswang1029** [comment]
Do you have detailed steps that I can follow to reproduce the issue?

**Comment by kswang1029** [comment]
nevermind, I think I found the way to reproduce the problem. Thanks for spotting this.

**Comment by kswang1029** [comment]
@keflavich please still provide detailed steps that I can try to follow in order to reproduce the issue. Thanks.

**Comment by kswang1029** [comment]
At the moment the only way I can reproduce the issue is to have images with different sky coverages so that the selected line region is totally outside the reference image, as demonstrated in the following screenshot. Do you think this is the case of yours?

![Screen Shot 2022-05-21 at 12 56 00](https://user-images.githubusercontent.com/20819712/169636070-0e741836-6057-4873-b116-c3c59f50f1b9.png)

**Comment by keflavich** [comment]
No, that wasn't the case for me - I had several panels open, but all were from the same EB and therefore had identical FOVs.

I will try to reproduce this and post more information.

**Comment by kswang1029** [comment]
@keflavich do you think this is still an issue with v4-stable?

**Comment by keflavich** [comment]
I haven't had much chance to work with v4-stable yet, so I'm not sure.
